web/src/games/cardtable/CardContainer.tsx

```diff
@@ -47,10 +52,39 @@ const CardContainer: FunctionComponent<ICardContainerProps> = (props: ICardConta
   const [isOpen, setOpen] = React.useState(false);
   const [selectedValue, setSelectedValue] = React.useState(19);
 
-  const handleClick = (evt: React.MouseEvent, idx?: number) => {
+  const isClickAllowed = (parentContainer: string): boolean => {
+    if (stage && parentContainer) {
+      switch (stage) {
+        case 'cuttingForDeal': {
+          return name === 'Deck'; //&& currentPlayer === eventClicker;
+        }
+        case 'thePlay': {
+          return (name === 'North Hand' && playerID === '0') || (name === 'South Hand' && playerID === '1');
```

Comment:
Can you use an enum here too instead of a string? Comparing string values isn't type safe and very error prone for typos and refactors

Reply:
added enum logic in next commit.